requirements.txt

```diff
@@ -0,0 +1 @@
+slack-webhook
```

Comment:
I don't use `requirements.txt` for this project, rather `pipenv`. Could it be possible to use `pipenv install slack-webhook` or put it in the `Pipfile` instead? Thank you :)

Reply:
It seems I'm living in the past! Switching to pipenv pronto 🚀